src/sgame/sg_main.cpp

```diff
@@ -365,8 +366,10 @@ static cvarTable_t gameCvarTable[] =
 
 	// gameplay: basic
 	{ &g_timelimit,                   "timelimit",                     "45",                               CVAR_SERVERINFO,                                 0, true     , nullptr       },
-	{ &g_friendlyFire,                "g_friendlyFire",                "1",                                CVAR_SERVERINFO,                                 0, true     , nullptr       },
-	{ &g_friendlyBuildableFire,       "g_friendlyBuildableFire",       "1",                                CVAR_SERVERINFO,                                 0, true     , nullptr       },
+	{ &g_friendlyFireAlienMultiplier, "g_friendlyFireAlienMultiplier", "0.5",                              0,                                 0, true     , nullptr       },
```

Comment:
Please use the C++ cvars (e.g. `Cvar::Cvar<float>`) for new ones

Reply:
Maybe it makes more sense to convert them in a latter PR so that they are updated in bulk?

Reply:
It's not like this is the first time C++ cvars are being introduced in the sgame. There are many new-style cvars already, so just old ones that haven't been converted are here.

Reply:
updated